src/connector/src/parser/mod.rs

```diff
@@ -937,6 +937,11 @@ impl ByteStreamSourceParserImpl {
             _ => unreachable!(),
         }
     }
+
+    /// Create a parser for testing purposes.
+    pub fn create_for_test(parser_config: ParserConfig) -> ConnectorResult<Self> {
```

Comment:
Why not `new_for_test`?

Reply:
To be consistent with the method above (`create`).